### ISSUE TITLE
About the grip on Vdrift Cars

### ISSUE THREAD

**Issue by leillo1975** [VDrift/vdrift#179]
Hello
Yesterday, I'm playing again with this great game and I was trying some cars in the new Algarve track. I felt that the most of cars has problems on grip when you brake the car, and is very easy to skid with oversteer when you come to a curve, even using ABS and pressing with caution the brake (at 20%). 
Finally I could complete some laps with P34 and F1-02, but this cars have a lot of grip because they use wings and has a lot of aerodinamic load.

Is possible to add more grip to the most of cars?  Where can I change this parameter?

**Comment by leillo1975** [comment]
No one here?

**Comment by leillo1975** [comment]
I'm sorry for insisting, can anyone give me an answer?

**Comment by franzcor** [comment]
This is true expecially for fast cars. Almost impossible to drive.. anything I can do to set the grip parameter?

**Comment by leillo1975** [comment]
@logzero , are you doing something with this in the last commits? Either I am now very good at driving or you have definitely done something with the grip at Vdrift. I drove the Ferrari F40 around Monza yesterday and it's much better than before.
